monitoring/files/http.conf

```diff
@@ -0,0 +1,59 @@
+server {
+    listen [::]:80 ipv6only=off;
+    server_name /;
+
+    location / {
+        return 301 https://$host$request_uri;
+    }
+}
+
+upstream grafana {
+    server grafana;
+}
+
+server {
```

Comment:
If we end up using an alb for ssl termination, we might not need to deploy nginx at all